functions/yum/repo/sanitize_simp_release_slug.pp

```diff
@@ -9,6 +9,8 @@ function simp::yum::repo::sanitize_simp_release_slug(
   Variant[String,Undef] $simp_release_slug = undef
 ) {
 
+  inspect(simplib::simp_version())
```

Comment:
Is there a reason this is not named spaced?  Should it be simplib::inspect?  I could not find a puppet core function called inspect.

Reply:
This was a mistake. Good catch!